### PR TITLE
Map set objects tests

### PR DESCRIPTION
**Pull request by tong-lambda**

### CODE REVIEW

File: test/map-set-objects.spec.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    sayings.delete("dog");
+    sayings.delete(1);
+    assert.strictEqual(sayings.has("dog"), false);
+    assert.strictEqual(sayings.has(1), false);
+    sayings.clear();
+    assert.strictEqual(sayings.size, 0);
+  });
+});
+
+describe("Set", function () {
```
Comment (josh-egan) [inline test/map-set-objects.spec.js:25]: The primary use of `Set` in practical programming is to get unique values. I would recommend adding a test case along those lines. `new Set([1,1,1,2,2,2,3,3,3,4,4,4,4,4,4,5])`. You will find in many places in our code something like `const uniqueVals = Array.from(new Set(vals));`

File: test/map-set-objects.spec.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    Intern.sayHi = sayHi;
+
+    assert.strictEqual(Manager.sayHi(), "Hello, my name is John.");
+    assert.strictEqual(Intern.sayHi(), "Hello, my name is Ben.");
+  });
+
+  it("Should use getter and setter", function () {
+    let o = {
+      a: 7,
+      get b() {
```
Comment (josh-egan) [inline test/map-set-objects.spec.js:117]: getters and setters are good to remember. They're useful particularly in Class style programming. You'll find them used frequently in `atlas-db`

File: test/map-set-objects.spec.js (hunk 1 above)
Comment (tong-lambda) [reply test/map-set-objects.spec.js:25]: Added test

**Review by josh-egan — APPROVED** [review]
:rocket: :100:

**Pull request merged**